sdl_viewer/src/bin/sdl_viewer.rs

```diff
@@ -20,9 +20,11 @@ use sdl_viewer::SdlViewer;
 fn main() {
     SdlViewer::new()
         .register_octree_factory("grpc:".into(), |p| {
+            let octree_data_provider = point_viewer_grpc::GrpcOctreeDataProvider::new(p.as_str())
+                .expect("Could not connect to GRPC server.");
             Ok(Box::new(
-                point_viewer_grpc::GrpcOctree::new(p.as_str())
-                    .expect("Could not connect to GRPC server"),
+                point_viewer::octree::Octree::new(Box::new(octree_data_provider))
+                    .expect("Could not create octree."),
```

Comment:
you could call `octree_from_address()` instead

Reply:
Done.